src/runtime_config/gov.lua

```diff
@@ -2,32 +2,48 @@
 -- Licensed under the Apache 2.0 License.
 
 -- This file defines the default initial contents (ie, Lua scripts) of the gov_scipts table.
-return {
-  quorum = [[
-  tables, calls = ...
-  -- count active members
-  n_active = 0
+return {  
+  pass = [[
+  tables, calls, votes = ...
+
+  member_votes = 0
+
+  for member, vote in pairs(votes) do
+    if vote then
+      member_votes = member_votes + 1
+    end
+  end
+
+  -- count active members, excluding operators
```

Comment:
This comment only applies to the operator version of the constitution.

Reply:
Yes!